### PR TITLE
Update requirements PyTAML and importlib_resources

### PR DESCRIPTION
**Pull request by lrodri29**
Just updating some of the listed python module reqs

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,10 @@
 boto3>=1.4.3
 botocore>=1.4.93
 hvac>=0.8.1
-PyYaml
+PyYaml>=5.3
```
Comment (movestill) [inline requirements.txt]: Let's use `>=5.2` since that's known to work and currently `awscli` specifies `<5.3`.

File: requirements.txt [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 mysql-connector-python
 pyminifier
 funcparserlib
 iso8601
 colorama
+importlib_resources
```
Comment (movestill) [inline requirements.txt]: Derek's heaviside `requirements.txt` specifies this like so:
`importlib_resources; python_version < "3.7"`

Comment (movestill) [reply requirements.txt]: Actually, maybe we want to make soft link to the `requirements.txt` in the heaviside submodule folder so it's clear that certain dependencies are only included due to that submodule.

`heaviside-requirements.txt -> lib/heaviside.git/requirements.txt`

Comment (sandyhider) [reply requirements.txt]: I like that approach too.

Comment (lrodri29) [reply requirements.txt]: You got it!

**Pull request merged**